src/web/app/pages-instructor/instructor-session-individual-extension-page/individual-extension-date-modal/individual-extension-date-modal.component.ts

```diff
@@ -50,21 +58,21 @@ export class IndividualExtensionDateModalComponent {
 
   extendByDeadlineKey: String = '';
   extendByDeadlineOptions: Map<String, Number> = new Map([
-    ['12 hours', 12],
-    ['1 day', 24],
-    ['3 days', 72],
-    ['1 week', 168],
-    ['Customize', 0],
+    ['12 hours', Hours.TWELVE],
+    ['1 day', Hours.IN_ONE_DAY],
+    ['3 days', Hours.IN_THREE_DAYS],
+    ['1 week', Hours.IN_ONE_WEEK],
+    ['Customize', Hours.ZERO],
   ]);
   extendByDatePicker = { hours: 0, days: 0 };
 
-  ONE_MINUTE_IN_MILLISECONDS = 60 * 1000;
-  ONE_HOUR_IN_MILLISECONDS = 60 * this.ONE_MINUTE_IN_MILLISECONDS;
-  ONE_DAY_IN_MILLISECONDS = 24 * this.ONE_HOUR_IN_MILLISECONDS;
+  ONE_MINUTE_IN_MILLISECONDS = Seconds.IN_ONE_MINUTE * Milliseconds.IN_ONE_SECOND;
+  ONE_HOUR_IN_MILLISECONDS = Minutes.IN_ONE_HOUR * this.ONE_MINUTE_IN_MILLISECONDS;
+  ONE_DAY_IN_MILLISECONDS = Hours.IN_ONE_DAY * this.ONE_HOUR_IN_MILLISECONDS;
```

Comment:
Could you help replace these? Shouldn't need these anymore with the new constants.